tr/po: End of line within string

dash-to-panel cannot be built right now, there's a missing closing quote.

```
glib-compile-schemas ./schemas/
msgfmt -c po/hu.po -o po/hu.mo
msgfmt -c po/tr.po -o po/tr.mo
msgfmt -c po/de.po -o po/de.mo
msgfmt -c po/es.po -o po/es.mo
msgfmt -c po/pl.po -o po/pl.mo
msgfmt -c po/fr.po -o po/fr.mo
msgfmt -c po/cs.po -o po/cs.mo
msgfmt -c po/uk.po -o po/uk.mo
msgfmt -c po/pt_BR.po -o po/pt_BR.mo
msgfmt -c po/kk.po -o po/kk.mo
msgfmt -c po/ru_RU.po -o po/ru_RU.mo
msgfmt -c po/zh_CN.po -o po/zh_CN.mo
po/tr.po:313: end-of-line within string
msgfmt: found 1 fatal error
make: *** [Makefile:53: po/tr.mo] Error 1
make: *** Waiting for unfinished jobs....
```

https://github.com/home-sweet-gnome/dash-to-panel/blob/d2a104e3eaf5895ea7e06b5a2b6f6c44f596d211/po/tr.po#L310-L313

--- a/po/tr.po
+++ b/po/tr.po
@@ -309,7 +309,7 @@ msgid ""
 "window peeking mode."
 msgstr ""
 "Bir pencere önizleme üzerinde vurgulama sırasında etkinlik dışı zaman girmek için gerekli "
-"pencere göz atma modu
+"pencere göz atma modu"
 
 #: Settings.ui.h:51
 msgid "Window peeking mode opacity"

